src/vs/workbench/services/configurationResolver/common/variableResolver.ts

```diff
@@ -26,6 +27,7 @@ export interface IVariableResolveContext {
 	getWorkspaceFolderPathForFile?(): string | undefined;
 	getSelectedText(): string | undefined;
 	getLineNumber(): string | undefined;
+	getExtension(id: string): Promise<{ readonly extensionLocation: uri } | undefined>;
```

Comment:
Should this be `getExtensionDir` instead?

Reply:
I named it like this in case we pull more stuff from the extension info at a later date. The return value from this is a subset of the data from `IExtensionService.getExtension`.